app/controllers/jira/status/push_controller.rb

```diff
@@ -26,12 +28,24 @@ class PushController < ApplicationController
           JiraIssuesAndPushes::ERROR_BLANK_LONG_RUNNING_MIGRATION.to_s => 'Migrations field is blank'
         }
       }.freeze
-
       before_action :find_sha_resources,    except: [:branch, :summary]
       before_action :find_branch_resources, only:   [:branch]
 
       def edit; end
 
+      def deploy_email
+        text =
+          if @push.email_sent?
+            'Email was already sent.'
```

Comment:
Feels like a better experience to show the table of items for this SHA? Easy way for that right now would be to redirect to the existing 'edit' page (and ideally with this text as a flash message -- not sure if we have plumbed the session.flash to the UI layout automatically yet though)

Reply:
Agreed. I'm going to change it to redirect to the edit page.